src/pages/atomicdex/api/v20-dev/non_fungible_tokens/index.mdx

```diff
@@ -0,0 +1,762 @@
+# Non Fungible Tokens (NFTs)
+
+The AtomicDEX API supports [ERC1155](https://www.nftstandards.wtf/Standards/ERC1155+Multi+token) and [ERC721](https://www.nftstandards.wtf/Standards/ERC721+Non+Fungible+Standard) NFTs via the [Moralis API](https://docs.moralis.io/) on the Avalanche (AVAX), BNB Smart Chain (BNB), Ethereum (ETH), Fantom (FTM), Polygon (MATIC) networks.
+
+
+
+## NFT Information Methods
+
+- Get a list of your tokens with [get_nft_list](/atomicdex/api/v20-dev/non-fungible-tokens/get-nft-list)
+- Get a list of token transfers with [get_nft_transfers](/atomicdex/api/v20-dev/non-fungible-tokens/get-nft-transfers)
+- Get token metadata with [get_nft_metadata](/atomicdex/api/v20-dev/non-fungible-tokens/get-nft-metadata)
+
+
+## NFT Transaction Methods
+
+- Withdraw ERC721 tokens with [withdraw_nft](/atomicdex/api/v20-dev/non-fungible-tokens/withdraw-nft#erc721)
+- Withdraw ERC1155 tokens with [withdraw_nft](/atomicdex/api/v20-dev/non-fungible-tokens/withdraw-nft#erc1155)
+
+
+## Get a list of NFTs {{label : 'get_nft_list', tag : 'API-v2'}}
+
+Returns a list of the NFTs owned by the user, shown in descending order of the `block_number` value (the block height when the amount or owner changed). If the request is for NFTs on more than one chain, this means that the order may not be chronological.
+
+### Request Parameters
+
+| Parameter     | Type             | Description                                                                                               |
+| ------------- | ---------------- | ----------------------------------------------------------------------------------------------------------|
+| chains        | array            | List of chains to scan for NFTs.                                                                          |
+| max           | boolean          | Optional, defaults to `false`. If `true`, response will return all NFTs without pagination, and will ignore the `limit` and `page_number` values.               |
+| limit         | integer          | Optional, defaults to `10`. The number of NFTs displayed per page in response.                            |
+| page_number   | integer          | Optional, defaults to `0`. The page offset for items in response.                                         |
+
+
+### Response Parameters
+
+| Parameter            | Type             | Description                                                                                      |
```

Comment:
[nft list info](https://gist.github.com/laruh/248d652d88891f3cadb09fea3a441466#get_nft_list-request) 

Some updates:
In `struct Nft` and `struct NftCommon` you can see that a lot of fields have become optional. 
Please note that `NftCommon` will be flattened and in response in Nft object there will be shown all fields from `NftCommon` instead of `common` field.

Also `collection_name` field right now called just `name`, as well as `token_name` inside `uri_meta`. it is also called `name`.
That is, the external `name` is the name of the collection, the `name` field inside `UriMeta` is the token name.

In addition, notes about `"image"` and `"image_url"` from `"uri_meta"`.
Right now `image` is the original field from `"token_uri"` link.
`"image_url"` is the field where we collect images from `token_uri` and `metadata` fields. If `image` or `image_url` are null in `token_uri` link, then we try to get one of them from `metadata`. 

Another note:
`token_uri` and `metadata` usually contain either `image` or `image_url` with image url. But most often nft creators use only `image` name for this value (from my observation), less often they use both parameters with the same url.